### PR TITLE
[docs] update npm run docs

### PR DESCRIPTION
**Pull request by bmorelli25**
* Fixes https://github.com/elastic/apm-agent-nodejs/issues/1072 by using docker and asciidoctor to build the docs.
* Fixes https://github.com/elastic/apm-agent-nodejs/issues/1071 by removing the docs build from travis

I was bored last night so I played around with this. I think I need your help, not sure what the best way to have a "build" and an "open" script is. In this potential solution I added a `docs:open` script that opens the documentation and a `docs:build` script that just builds it. Not sure what a best practice would be though.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "types": "index.d.ts",
   "scripts": {
-    "docs": "./docs/scripts/build_docs.sh apm-agent-nodejs ./docs ./build",
```
Comment (watson) [inline package.json]: We have a Travis test that currently expects this script to be there:

https://github.com/elastic/apm-agent-nodejs/blob/9abb4aa926fdca65e70c570735f8f91985905952/.travis.yml#L63-L67

So we should either update that or remove the build from Travis, which would also fix #1071

Comment (bmorelli25) [reply package.json]: Ah, thanks, forgot we talked about this as well!

File: package.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
...
+    "docs:open": "PREVIEW=1 ./docs/scripts/build_docs.sh apm-agent-nodejs ./docs ./build",
```
Comment (Qard) [inline package.json]: I believe you could also do `PREVIEW=1 npm run docs:build` to reduce duplication.

Comment (bmorelli25) [reply package.json]: oooo I like that! Let me try

Comment (bmorelli25) [reply package.json]: Nice, it works!

**Comment by bmorelli25** [comment]
I'll let one of you look at the failing travis build and decide if this is ok to merge or not

**Comment by watson** [comment]
It's a bug that's fixed in master, but unrelated to this PR, so I'll go ahead and merge 😊

**Pull request merged**